string constellation
uint8 signal_strength